index.js

```diff
@@ -150,10 +150,11 @@ function mockServiceMethod(service, client, method, replace) {
 
     if ((client.config || _AWS.config).paramValidation) {
       try {
-        var inputRules = client.api.operations[method].input;
-        var outputRules = client.api.operations[method].output;
-        var params = userArgs[(userArgs.length || 1) - 1];
-        new _AWS.ParamValidator((client.config || _AWS.config).paramValidation).validate(inputRules, params);
+        var inputRules = (client.api.operations[method] || {}).input;
```

Comment:
Makes sense.  I was wondering if the recent `aws-sdk` update broke the `client.api` in this case.  It looks like it may have.  Thanks for the clarification.  We are reverting to `aws-sdk-mock@1.5.0` until we can find a resolution.